packages/cursorless-everywhere-talon/src/test/quickjsTest.ts

```diff
@@ -0,0 +1,93 @@
+import { getTestHelpers } from "talon";
+import { activateTest } from "cursorless.mjs";
+import type { ActionDescriptor, CommandLatest } from "@cursorless/common";
+
+async function runTests() {
+  print("Running quickjs tests");
+
+  await activateTest();
+
+  await test("testTake", testTake);
+  await test("testChuck", testChuck);
+}
+
+async function testTake() {
+  const testHelpers = getTestHelpers();
+  const initialText = "Hello, world!";
+
+  testHelpers.setEditorState({
+    text: initialText,
+    selections: [{ anchor: 0, active: 0 }],
+  });
+
+  await runAction({
+    name: "setSelection",
+    target: {
+      type: "primitive",
+      modifiers: [{ type: "containingScope", scopeType: { type: "token" } }],
+    },
+  });
+
+  const { text, selections } = testHelpers.getFinalEditorState();
+
+  assert.equal(text, initialText);
+  assert.equal(selections.length, 1);
+  assert.equal(selections[0].anchor, 0);
+  assert.equal(selections[0].active, 5);
+}
+
+async function testChuck() {
+  const testHelpers = getTestHelpers();
+
+  testHelpers.setEditorState({
+    text: "Hello, world!",
+    selections: [{ anchor: 0, active: 0 }],
+  });
+
+  await runAction({
+    name: "remove",
+    target: {
+      type: "primitive",
+      modifiers: [{ type: "containingScope", scopeType: { type: "token" } }],
+    },
+  });
+
+  const { text, selections } = testHelpers.getFinalEditorState();
+
+  assert.equal(text, ", world!");
+  assert.equal(selections.length, 1);
+  assert.equal(selections[0].anchor, 0);
+  assert.equal(selections[0].active, 0);
+}
+
+function runAction(action: ActionDescriptor) {
+  const command: CommandLatest = {
+    version: 7,
+    usePrePhraseSnapshot: false,
+    action,
+  };
+  return getTestHelpers().contextActions.private_cursorless_run_rpc_command_get(
+    "cursorless.command",
+    command,
+  );
```

Comment:
Here, for example, I'm not following why you can't just import actions from talon and run the action that way

Reply:
That is not on the actions type by design. Only the test runner should ever call it.